frontend/docs/angular.mdx

```diff
@@ -0,0 +1,107 @@
+---
+id: angular
+title: Angular
+hide_title: false
+hide_table_of_contents: false
+keywords:
+  - final space
+  - finalspace
+  - finalspaceapi
+  - final space wiki
+  - api
+  - restapi
+  - rest api
+  - example
+  - angular
+description: A live example of the Final Space API using Angular.
+---
+
+
+<Tabs
+  defaultValue="angular"
+  values={[
+    {label: 'Angular', value: 'angular'}
+  ]}>
+  <TabItem value="angular">
+
+```jsx
+{/* Not a Live Editor*/}
+
+{/* app.component.ts*/}
+import {Component} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent {
+  title = 'Final Space API with Angular';
+  url = 'https://finalspaceapi.com/api/v0/character?limit=5';
+  items = [];
+
+  constructor(private http: HttpClient) {
+    this.http.get(this.url).toPromise().then(data => {
+      console.log(`>>> data => `, data);
+
+      for (const key in data){
+        if (data.hasOwnProperty(key)){
+          this.items.push(data[key]);
+        }
+      }
+    });
+  }
+}
+
+{/* app.component.html*/}
```

Comment:
Can you separate it into two code snippets 
```
{/* app.component.ts*/}
```

```
{/* app.component.html*/}
```

Also for css

Reply:
ok